gossip/c_block_callbacks.go

```diff
@@ -222,22 +245,10 @@ func consensusCallbackBeginBlockFn(
 					// sort events by Lamport time
 					sort.Sort(confirmedEvents)
 
-					// new block
-					var block = &inter.Block{
-						Time:    blockCtx.Time,
-						Atropos: cBlock.Atropos,
-						Events:  hash.Events(confirmedEvents),
-					}
 					for _, tx := range append(preInternalTxs, internalTxs...) {
```

Comment:
I think with this change internal transactions get added to the end of the block instead of being placed at the head - right?

Reply:
This change changes how they are orderer in the block, but the internal transactions are still executed first. I thought only the execution of the transactions matter - fixed.

Reply:
Transactions from events are actually never appended to the same array as internal transactions.
Internal transactions are added [here](https://github.com/Fantom-foundation/Sonic/blob/729e04882b9e3efd16e9a3f2b36807c40ffacbf0/gossip/c_block_callbacks.go#L232). Event transactions are just executed. 
So I guess my assumption that only order of `Execute(...)` calls matter should be correct? Anyhow, I still rearranged the code to append the internal transactions first.